### ISSUE TITLE
Ticker.h: No such file or directory

### ISSUE THREAD

**Issue by sevketk** [witnessmenow/LED-Matrix-Display-Examples#3]
I setup Arduino for esp8266 board from https://github.com/esp8266/ from boards manager . and I connect my led panel directly to arduino (not use esp8266). Bu however ticker.h not found. When I compile banana code I get error. 

`Arduino: 1.8.10 (Windows 10), Board: "Arduino/Genuino Mega or Mega 2560, ATmega2560 (Mega 2560)"

arduino_rgb_ballgame:5:10: error: Ticker.h: No such file or directory

 #include <Ticker.h>

          ^~~~~~~~~~

compilation terminated.

exit status 1
Ticker.h: No such file or directory

This report would have more information with
"Show verbose output during compilation"
option enabled in File -> Preferences.
`
but the other hand , pixeltime_AVR example work for me (it says t me, my wiring , setup is correct)
Why dont work banana example?

**Comment by witnessmenow** [comment]
You have the wrong board selected in the Arduino IDE

**Comment by sevketk** [comment]
but the other hand , pixeltime_AVR example work for me (it says t me, my wiring , setup is correct)

https://github.com/2dom/PxMatrix/tree/master/examples/pixeltime_AVR

**Comment by witnessmenow** [comment]
Pixel time avr is written for avr, these are written for esp8266, maybe
they will work if ported but they are currently only coded for the esp8266

**Comment by sevketk** [comment]
Yes I have already carried. but this time "ticker.h" did not work.  I changed ports and connections. no problem. I didn't know what to replace "ticker.h". I tried to use "millis ()" function instead of "ticker.h" but I couldn't.
I would be grateful if you put the avr example for "banana dance".
